app/Http/Controllers/domainController.php

```
@@ -154,6 +154,39 @@ public function updateDomain(Request $request)
         }
     }
 
+    public function deleteDomain(Request $request, $link)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)